cmd/entrypoint/namespaces_test.go

```diff
@@ -0,0 +1,41 @@
+// +build linux
+
+package main
+
+import (
+	"os/exec"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+// This isn't a great unit test, but it's the best I can think of.
+// It attempts to verify there is no network access by making a network
+// request. If the test were to run in an offline environment, or an already
+// sandboxed environment, the test could pass even if the dropNetworking
+// function did nothing.
```

Comment:
i guess the other thing you could do is assert on the exact values you set on the cmd object but that doesn't feel super useful